README.md

```diff
@@ -1,2 +1,171 @@
-# talk-2022-juliacon_toolchain
-From Mesh Generation to Adaptive Simulation: A Journey in Julia at JuliaCon 2022
+# JuliaCon 2022: From Mesh Generation to Adaptive Simulation: A Journey in Julia
+
+[![License: MIT](https://img.shields.io/badge/License-MIT-success.svg)](https://opensource.org/licenses/MIT)
+
+<p align="center">
+  <a href="https://www.youtube.com/watch?v=hoViWRAhCBE" target="_blank" rel="noopener noreferrer"><img
+    src="https://user-images.githubusercontent.com/25242486/176432903-668ce8bf-4119-4d15-a46e-a1df90944e14.png"
+    width="500px" /></a>
+</p>
+
+This is the companion repository for the [JuliaCon 2022](https://juliacon.org/2022) talk
+
+**From Mesh Generation to Adaptive Simulation: A Journey in Julia**<br>
+*Andrew R. Winters*<br>
+
+(see abstract [below](#abstract)). Here you can find the presentation slides
+in [talk.pdf](talk.pdf) as well as Julia scripts in the [examples](examples/)
+directory to locally create results of mesh generation with
+[HOHQMesh.jl](https://github.com/trixi-framework/HOHQMesh.jl)
+and [Trixi.jl](https://github.com/trixi-framework/Trixi.jl) simulations
+presented in the talk.
+
+To reduce the size of the file for the [talk.pdf](talk.pdf), the video is not
+embedded. Instead, the video `cylinder_flow_sine_walls.mp4` shown in the
+presentation is located in [media](media/).
```

Comment:
The movie doesn't play in browser due to size. If you have a Youtube link, might be good to add here as well.

Reply:
That is a good point. I plan to upload the video to YouTube the week of JuliaCon, so i can add a link after that.

Reply:
I opened an issue #2 to remind me to add the YouTube links

Reply:
It is on YouTube now and I will add the link. Should I remove the actual movie file from the repo?

Reply:
I think yeah, saves some space.